app/reviews/models.py

```diff
@@ -1,5 +1,34 @@
 from __future__ import unicode_literals
 
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+from common.mixins import TimestampMixin
+from app.timetables.models import Serving
+
+
+class Review(TimestampMixin):
+    """
+    Model representing ratings and comments
+    submitted by users for menu item
+    they had on a particular day of the cycle.
+    """
+
+    RATINGS = (
+        (5, 'EXCELLENT'),
+        (4, 'GOOD'),
+        (3, 'FAIR'),
+        (2, 'POOR'),
+        (1, 'TERRIBLE')
+    )
```

Comment:
According to Django documentation, it’s best to define choices inside a model class, and to define a suitably-named constant for each value.
Though you can define a choices list outside of a model class and then refer to it, defining the choices and names for each choice inside the model class keeps all of that information with the class that uses it, and makes the choices easy to reference.

So we can have something like this instead 
```
class Student(models.Model):
    FRESHMAN = 'FR'
    SOPHOMORE = 'SO'
    JUNIOR = 'JR'
    SENIOR = 'SR'
    YEAR_IN_SCHOOL_CHOICES = (
        (FRESHMAN, 'Freshman'),
        (SOPHOMORE, 'Sophomore'),
        (JUNIOR, 'Junior'),
        (SENIOR, 'Senior'),
    )
```
e.g, Student.SOPHOMORE will work anywhere that the Student model has been imported.

https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.Field.choices